README.md

```diff
@@ -432,16 +432,16 @@ Validation for simple data types (e.g. integers, floating point numbers, strings
   * `maximumValueExclusive`: Reject values that are greater than or equal to this. No restriction by default.
 * `boolean`: The value is either `true` or `false`. No additional parameters.
 * `datetime`: The value is a simplified [ECMAScript ISO 8601](https://www.ecma-international.org/ecma-262/5.1/#sec-15.9.1.15) date string with optional time and time zone components (e.g. "2016-06-18T18:57:35.328-08:00"). If both time and time zone are omitted, the time is assumed to be midnight UTC. If a time is provided but the time zone is omitted, the time zone is assumed to be the Sync Gateway server's local time zone. Additional parameters:
-  * `minimumValue`: Reject date/times that are less than this. May be either an ECMAScript ISO 8601 date string with optional time and time zone components OR a JavaScript [`Date`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) object. The time is assumed to be midnight if the time component is omitted and the time zone is assumed to be UTC if the time zone component is omitted. No restriction by default.
-  * `minimumValueExclusive`: Reject date/times that are less than or equal to this. May be either an ECMAScript ISO 8601 date string with optional time and time zone components OR a JavaScript `Date` object. The time is assumed to be midnight if the time component is omitted and the time zone is assumed to be UTC if the time zone component is omitted. No restriction by default.
-  * `maximumValue`: Reject date/times that are greater than this. May be either an ECMAScript ISO 8601 date string with optional time and time zone components OR a JavaScript `Date` object. The time is assumed to be midnight if the time component is omitted and the time zone is assumed to be UTC if the time zone component is omitted. No restriction by default.
-  * `maximumValueExclusive`: Reject date/times that are greater than or equal to this. May be either an ECMAScript ISO 8601 date string with optional time and time zone components OR a JavaScript `Date` object. The time is assumed to be midnight if the time component is omitted and the time zone is assumed to be UTC if the time zone component is omitted. No restriction by default.
```

Comment:
The bit about default values for missing time and time components has been removed because it is technically inaccurate and already addressed above.